sources/en/r/relibrary.py

```diff
@@ -0,0 +1,58 @@
+import logging
+from typing import Generator
+
+from bs4 import BeautifulSoup, Tag
+from time import sleep
+
+from lncrawl.models import Chapter
+from lncrawl.templates.soup.chapter_only import ChapterOnlySoupTemplate
+
+logger = logging.getLogger(__name__)
+
+
+class relibCrawler(ChapterOnlySoupTemplate):
+    base_url = ["https://re-library.com/"]
+
+    def initialize(self) -> None:
+        self.cleaner.bad_css.update(
+            [
+                ".nextPageLink",
+                ".prevPageLink",
+                ".su-button",
+                "a[href*='re-library.com']",
+            ]
+        )
+        self.cleaner.bad_tag_text_pairs.update({"h2": "References"})
```

Comment:
To reduce job count: `self.init_executor(<your job count here>)`

Reply:
Thanks, I did not find the way to do it

Reply:
Even with 1 worker, the rate detection system is still firing..., is there anyway to force the maximum rate of item/s?

Reply:
It is difficult to determine a proper rate since we don't know exactly how their rate detection works. I do not want to sacrifice the download speed over the chance to get IP blocked. People using proxy will not face this issue. And there is also Google Colab.